financial-connections/src/main/java/com/stripe/android/financialconnections/image/StripeImage.kt

```diff
@@ -0,0 +1,84 @@
+package com.stripe.android.financialconnections.image
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.BoxWithConstraintsScope
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.graphics.painter.BitmapPainter
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.unit.Dp.Companion.Infinity
+import androidx.compose.ui.unit.IntSize.Companion.Zero
+import com.stripe.android.financialconnections.ui.LocalImageLoader
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+
+@Composable
+internal fun StripeImage(
+    url: String,
+    placeHolder: Painter,
+    modifier: Modifier = Modifier,
+    contentDescription: String? = null
+) {
+    val imageLoader = LocalImageLoader.current
+    BoxWithConstraints(modifier) {
+        val (width, height) = calculateBoxSize()
+        val painter: MutableState<Painter> =
+            remember { mutableStateOf(BitmapPainter(ImageBitmap(width, height))) }
+        var loadImageJob: Job?
+        val scope = rememberCoroutineScope()
+
+        DisposableEffect(url) {
+            loadImageJob = scope.launch(Dispatchers.Default) {
```

Comment:
Nit: Should this be `Dispatchers.IO` instead?

Reply:
https://github.com/stripe/stripe-android/pull/5307/commits/a720ee0b4954bcc445693d462a47042d5a0a42ad